maintenance_timesheet/views/maintenance_request_views.xml

```diff
@@ -0,0 +1,158 @@
+                    groups="maintenance.group_equipment_manager"
+                    widget="timesheet_uom"
+                />
+            </field>
+        </field>
+    </record>
+    <record id="hr_equipment_request_view_kanban" model="ir.ui.view">
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='archive']" position="before">
+                <field
+                    name="timesheet_total_hours"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                />
+            </xpath>
+            <xpath expr="//footer//field[@name='activity_ids']" position="after">
+                <a
+                    class="o_project_kanban_box"
+                    name="action_view_timesheet_ids"
+                    type="object"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                >
+                    <div>
+                        <span class="o_value">
+                            <field
+                                name="timesheet_total_hours"
+                                class="oe_inline"
+                                widget="timesheet_uom"
+                            />
+                        </span>
+                        <span class="oe_label">Hours</span>
+                    </div>
+                </a>
+            </xpath>
+        </field>
+    </record>
+    <record id="hr_equipment_request_view_form" model="ir.ui.view">
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="base_maintenance.equipment_request_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button
+                    class="oe_stat_button"
+                    type="object"
+                    name="action_view_timesheet_ids"
+                    icon="fa-calendar"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                >
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value">
+                            <field
+                                name="timesheet_total_hours"
+                                widget="timesheet_uom"
+                            />
+                        </span>
+                        <span class="o_stat_text">
+                            Hours
+                        </span>
+                    </div>
+                </button>
+            </xpath>
+        </field>
+    </record>
+    <record id="equipment_request_view_form" model="ir.ui.view">
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="base_maintenance.equipment_request_view_form" />
+        <field name="arch" type="xml">
```

Comment:
```suggestion
        </field>
    </record>
    <record id="equipment_request_view_form" model="ir.ui.view">
        <field name="model">maintenance.request</field>
        <field name="inherit_id" ref="base_maintenance.equipment_request_view_form" />
        <field name="arch" type="xml">
```
There are 2 different views in this module (hr_equipment_request_view_form + equipment_request_view_form) that inherit from the same view (`base_maintenance.equipment_request_view_form`) so you can leave everything in one.